tests/sentry/tasks/test_commit_context.py

```diff
@@ -996,6 +998,36 @@ def test_gh_comment_pr_too_old(self, get_jwt, mock_comment_workflow, mock_get_co
             assert not mock_comment_workflow.called
             assert len(PullRequestCommit.objects.all()) == 0
 
+    @patch("sentry.integrations.github.client.get_jwt", return_value=b"jwt_token_1")
+    @responses.activate
+    def test_gh_comment_pr_info_level_issue(
+        self, get_jwt, mock_comment_workflow, mock_get_commit_context
+    ):
+        """No comment on pr that's has info level issue"""
+        mock_get_commit_context.return_value = [self.blame]
+        self.pull_request.date_added = before_now(days=1)
+        self.pull_request.save()
+
+        self.add_responses()
+
+        group = Group.objects.get_from_cache(id=self.event.group_id)
+        Group.update(group, level=logging.INFO)
+
+        with self.tasks():
+            event_frames = get_frame_paths(self.event)
+            process_commit_context(
+                event_id=self.event.event_id,
+                event_platform=self.event.platform,
+                event_frames=event_frames,
+                group_id=self.event.group_id,
+                project_id=self.event.project_id,
+            )
+            assert not mock_comment_workflow.called
+            assert len(PullRequestCommit.objects.all()) == 0
+
+        # Reset group level
+        Group.update(group, level=logging.ERROR)
```

Comment:
nit: you don't need to do this because changes you apply to something in a test don't affect other tests!